packages/browser-extension/src/styles/dialog.css

```diff
@@ -227,3 +227,28 @@ report-dialog-submit-view {
 report-dialog-submit-view[hidden] {
   display: none;
 }
+
+textarea-report-dialog {
```

Comment:
Remember to remove this line after one of the change requests above.

Reply:
Done